news/2021-05-13-mappers-report-april.md

```diff
@@ -0,0 +1,172 @@
+
+Next up, the April edition of Monthly Beatmapping Contest greets us with a few novelties. For the first time ever, the contest will entertain osu!catch submissions in addition to osu! and osu!taiko. Though the submission period has since ended, the limitations for April were as follows:
+
+- **osu!**:
+    - All sliders must use red anchors.
+    - Red anchors must form clear corners in the slidershape.
+- **osu!taiko**:
+    - No more than 3 of the same note colours in a row.
+- **osu!catch**:
+    - Sliders cannot be used.
+
+If you're excited to see the results, we suggest you take few minutes to look through [this newspost](https://osu.ppy.sh/home/news/2021-04-08-monthly-beatmapping-contest-april-2021) to keep yourself up-to-date with the details.
+
+For those unfamiliar with the format of the Monthly Beatmapping Contest and its proceedings, go ahead and have a read of the [wiki article](/wiki/Contests/Monthly_Beatmapping_Contest).
+
+### osu! Beatmapping World Championship 2021: Registrations Open
+
+It's that time of the year again! As for the third mapping contest of this issue, we're featuring the third iteration of the osu! Beatmapping Championship, registrations for which have recently opened.
+
+Keeping the notion of country-based competition, the 2021 edition of the contest is allowing multiple teams to be formed to represent a particular country, yet only the best one among them will get the chance of proceeding further than the first round. May the best team win!
+
+What's more, a judging panel featuring a notable batch of professional player circle-clickers has been added to the contest proceedings, which will function like a side dish to the main contest. As a hit squad of high-level mappers are evaluating the submissions, the professional player panel will be testplaying these to rate them in accordance with a certain set of criteria. Sounds interesting, doesn't it?
+
+Oh, if you're a drum basher, fruit catcher or keyboard masher, this one is just for you! This year, the osu!taiko, osu!catch and osu!mania editions of Beatmapping World Championship are going to be organized as well! The details for these contests are work-in-progress, but you still have a chance to partake in the judging panels if you're interested in helping out - check out [this form](https://forms.gle/Yu2M2ZiwhapJYc7K7) for more details!
+
+The particulars concerning the contest and so much more can be found in full within [this newspost](https://osu.ppy.sh/home/news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open), the [wiki article](/wiki/Contests/oBWC/3) and the [official website](https://obwc.net/). Check them all out!
+
+### osu!mania J-Rock Mapping Contest
+
+Attention to all J-Rock fans out there! If you're keen on playing a keytar as well, this will probably catch your interest.
```

Comment:
"as well" doesn't connect to anything

```suggestion
Attention to all J-Rock fans out there! If you're keen on playing the keytar, this will probably catch your interest.
```